src/language-css/printer-postcss.js

```diff
@@ -582,7 +582,19 @@ function genericPrint(path, options, print) {
           isNextForKeyword ||
           isNextEachKeyword
         ) {
-          parts.push(line);
+          if (atRuleAncestorNode && atRuleAncestorNode.name === "import") {
+            if (iNextNode.open) {
+              parts.push(iNextNode.open.raws.before.replace(/\s\s+/g, " "));
+            } else if (iNode.close || (iNode.group && iNode.group.close)) {
+              parts.push(line);
+            } else if (iNextNode && iNextNode.raws) {
+              parts.push(iNextNode.raws.before.replace(/\s\s+/g, " "));
+            } else if (iNode && iNode.raws) {
+              parts.push(iNode.raws.after.replace(/\s\s+/g, " "));
+            }
```

Comment:
Looks very crazy, we should:
1. Search other solution
2. If no solution found add comments about each `if`

Even I can not understand what is happening here :confused:

Reply:
@evilebottnawi I added this code when I tried fixing comments and even I didn't like this code . But I am removing it now as you said to keep comments as it is and adding url check https://github.com/prettier/prettier/pull/4435/commits/ff3d67ff2d49e24351b8b3aa490edcac8cf45c9e